lighthouse-core/scripts/update-report-fixture.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
I was thinking of checking in the latest-run artifacts folder for this fixture. so we'd have 1 script to run -A against it... and a second to refresh the artifacts. hows that sound?

Reply:
so if I understand your proposal, is it two scripts

`scripts/run-dash-a-against-dbw.sh`
and
`scripts/update-fixtures-with-latest-artifacts.sh`

names pending? :) that sounds fine but not sure why we would necessarily need both. Is there a use case for running just the first?